web_widget_image_paste_clipboard/static/src/js/web_widget_image_paste_clipboard.js

```diff
@@ -0,0 +1,39 @@
+odoo.define('web_widget_image_paste_clipboard.FieldBinaryImage', function(require) {
+    "use strict";
+
+    var basicFields = require('web.basic_fields');
+
+    basicFields.FieldBinaryImage.include({
+        _render: function() {
+            this._super.apply(this, arguments);
+            var self = this;
```

Comment:
pls address this comment https://github.com/OCA/web/pull/726#discussion_r254575932